jkube-kit/config/service/src/test/java/org/eclipse/jkube/kit/config/service/kubernetes/KubernetesClientUtilTest.java

```diff
@@ -163,4 +162,58 @@ void getPodStatusMessagePostfix_whenActionIsError_shouldReturnErrorMessage() {
     assertThat(messagePostfix).isEqualTo(": Error");
   }
 
+  @Test
+  void getPodCondition_whenPodConditionIsReadyAndTrue_shouldReturnReady() {
+    // Given
+    Pod pod = new Pod();
+    PodStatus podStatus = new PodStatus();
+    PodCondition podCondition = new PodCondition();
+    podCondition.setType("ready");
+    podCondition.setStatus("True");
+    podStatus.setConditions(Collections.singletonList(podCondition));
+    pod.setStatus(podStatus);
+
+    // When
+    String condition = KubernetesClientUtil.getPodCondition(pod);
+
+    // Then
+    assertThat(condition).isEqualTo("ready");
+  }
+
+  @Test
+  void getPodCondition_whenPodConditionIsReadyAndFalse_shouldReturnEmptyString() {
+    // Given
+    Pod pod = new Pod();
+    PodStatus podStatus = new PodStatus();
+    PodCondition podCondition = new PodCondition();
+    podCondition.setType("ready");
+    podCondition.setStatus("False");
+    podStatus.setConditions(Collections.singletonList(podCondition));
+    pod.setStatus(podStatus);
+
+    // When
+    String condition = KubernetesClientUtil.getPodCondition(pod);
+
+    // Then
+    assertThat(condition).isEmpty();
+  }
+
+  @Test
+  void getPodCondition_whenPodConditionIsNotReady_shouldReturnEmptyString() {
+    // Given
+    Pod pod = new Pod();
+    PodStatus podStatus = new PodStatus();
+    PodCondition podCondition = new PodCondition();
+    podCondition.setType("notready");
```

Comment:
Could you please use a PodCondition from known values? I think possible values are listed here

https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-conditions